usr/share/rear/prep/GNU/Linux/300_include_grub_tools.sh

```diff
@@ -22,7 +22,8 @@ PROGS+=( grub-bios-setup      grub2-bios-setup
          grub-probe           grub2-probe
          grub-reboot          grub2-reboot
          grub-set-default     grub2-set-default
-         grub-setup           grub2-setup )
+         grub-setup           grub2-setup
+         grubby               grub2-editenv )
```

Comment:
I believe you should add also `grub-editenv` to preserve symmetry with the rest (and indeed, I checked a recent version of Debian, it has `grub-editenv`, not `grub-editenv`).